### ISSUE TITLE
Tizen platform support

### ISSUE THREAD

**Issue by listepo** [flutter/flutter#26152]
Hi, any plans for [tizen](https://www.tizen.org) support?

**Comment by MarkInMd** [comment]
I would love to be able to develop Tizen apps with Flutter.

**Comment by kangwang1988** [comment]
cc @timsneath

**Comment by Pingear** [comment]
+

**Comment by davystrong** [comment]
I would also really like Tizen support, specifically for the Gear/Galaxy Wearables.

**Comment by ivanooi** [comment]
me too!

**Comment by nowtech** [comment]
I have flutter apps in development that would suit the galaxy gear s3 so +1 for me.

**Comment by kroppli** [comment]
+1

**Comment by incker** [comment]
+1

**Comment by ping996** [comment]
does the flutter engine can run tizen system now?

**Comment by ping996** [comment]
does anyone how knows how to build arm32 linux type of flutter engine embedder library of libflutter_engine.so?

**Comment by sahandmax** [comment]
+1

**Comment by inaffect-ag** [comment]
any news on tizen support?

**Comment by bounmed** [comment]
any news on Tizen support?

**Comment by chertov** [comment]
+1

**Comment by ivanooi** [comment]
+2

**Comment by alereisan** [comment]
+3

**Comment by sajphon** [comment]
+4

**Comment by sheshnathiicmr** [comment]
+5

**Comment by sentialx** [comment]
+6

**Comment by davystrong** [comment]
With the progress in Flutter web, it seems to be quite easy to develop a web app for the Galaxy Watch. Obviously it's not as good as a native implementation but it at least allows us to build decent UI with ease. Doesn't work for watch faces though, but maybe this is enough for someone.

**Comment by fisforfaheem** [comment]
They better support UWP apps and PWA only first

**Comment by haipham22** [comment]
+7

**Comment by swift-kim** [comment]
I'm working on this. I was able to run some Flutter apps on Tizen TVs. The biggest problem is, it is not allowed to deploy native apps (not .NET or web apps) on TV devices due to security restrictions. It's not able to bypass them on customer devices as of now. I hope there will be a good news in the future.

**Comment by JosueMCapita** [comment]
+8

**Comment by Sanhajio** [comment]
There is this project by a person from Samsung, I am trying it: https://github.com/swift-kim/flutter-tizen

**Comment by swift-kim** [comment]
@Sanhajio Yes, the new URL is https://github.com/flutter-tizen/flutter-tizen.

The project is still not production ready (you cannot upload apps created with the tool to the store) but you can still try it for fun.

**Comment by vladislav1nikov** [comment]
Any news on Tizen support?

**Comment by MrBirb** [comment]
@vladislav1nikov [Samsung is porting Flutter to Tizen](https://github.com/flutter-tizen/flutter-tizen)
